src/prompt-storage.ts

```diff
@@ -0,0 +1,17 @@
+import Cookies from "js-cookie";
+
+export const markPromptMessageCompleted = (
+  userId: string,
+  promptId: string,
+  expiresAt: Date,
+) => {
+  Cookies.set(`prompt-${userId}`, promptId, { expires: expiresAt });
```

Comment:
We should probably be more obvious in the cookie naming, so it's easier for our customers developers to figure out where this cookie came from.
```suggestion
  Cookies.set(`bucket-prompt-${userId}`, promptId, { expires: expiresAt });
```